tests/test_fuzz.py

```diff
@@ -127,10 +139,7 @@ def test_multicast_sender_with_random_data(self, data):
 				)
 				self.assertIsNotNone(
 					multicast.__main__.McastDispatch().doStep("SAY", _fixture_SAY_args)
-				)
-				self.assertIsNotNone(
-					multicast.__main__.McastDispatch().doStep("SAY", _fixture_SAY_args)
-				)
+				)  # preemptive extra retry
```

Comment:
✅ LGTM. test already can be slow with some values sent over the network so reducing re-sends to two and adding an explanatory comment makes sense.